cypress/support/commands.ts

```diff
@@ -120,3 +119,12 @@ Cypress.Commands.add('skipSetup', () => {
 		});
 	});
 })
+
+Cypress.Commands.add('resetDatabase', () => {
```

Comment:
since this is resetting the db as well as the session, should we rename this to just `reset` ?